guardian/tag-janitor
# Add prettier to the cdk directory and format

## What does this change?

This PR adds a `.prettierrc` file to the `cdk` directory and runs the `format` script defined in `cdk/package.json`

## How to test

Run `yarn format` in the cdk directory again and see that it doesn't change anything this time

## How can we measure success?

All of the files in the `cdk` directory conform to the required style.
